devices/novo.js

```diff
@@ -7,9 +7,10 @@ const ea = exposes.access;
 module.exports = [
     {
         fingerprint: [{modelID: 'TS0601', manufacturerName: '_TZE200_swhwv3k3'}],
+        zigbeeModel: ['TS0601'],
```

Comment:
Please remove this line and check if it still works.

Reply:
done!